src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -28,14 +28,43 @@
      * 26.04.2019 Сергей 7 100
      * 26.04.2019 София 9 100
      * 26.04.2019 Сергей 11 50
-     *
+     * </p>
      * Пример вывода:
      * Отчёт за период 01.04.2019  - 30.04.2019
      * Сергей - 1550
      * Андрей - 600
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder allSalary = new StringBuilder();
+        int minData = Integer.parseInt(dateFrom.replaceAll("[^0-9]", ""));
```

Comment:
what if you will compare dateFrom = 14.07.2019 and dateTo = 10.08.2019?

Reply:
fixed